src/web/mjs/connectors/Cycomi.mjs

```diff
@@ -0,0 +1,61 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class CyComi extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'cycomi';
+        super.label = 'CyComi';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://cycomi.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const id = uri.pathname;
+        const title = (await this.fetchDOM(request, 'div.title-texts > h3'))[0].textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const mangaList = [];
+        for(let page = 0; page < 8; page++) {
```

Comment:
Looks like this needs to be incremented regualry, isn't it possible to use an abort criteria (e.g. page after last page does not contain mangas)?
See: https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/templates/%40Samples.mjs#L46-L56